ci/gpu/build.sh

```diff
@@ -100,6 +100,7 @@ else
     # Test with TCP/Sockets
     gpuci_logger "TEST WITH TCP ONLY"
     py.test --cache-clear -vs --ignore-glob tests/test_send_recv_two_workers.py tests/
+    py.test --cache-clear -vs ucp/_libs/tests
```

Comment:
Can these two lines be consolidated? (I haven't tested this suggestion)

```suggestion
    py.test --cache-clear -vs --ignore-glob tests/test_send_recv_two_workers.py tests/ ucp/_libs/tests/
```

Reply:
I tried, but pytest complains about name collision.